cmd/leases.go

```diff
@@ -103,9 +104,22 @@ var leasesEndCmd = &cobra.Command{
 
 var leasesLoginCmd = &cobra.Command{
 	Use:   "login [Lease ID]",
-	Short: "Login to a leased DCE account. (Sets AWS CLI credentials if used with no flags)",
-	Args:  cobra.ExactArgs(1),
+	Short: "Login to a leased DCE account. \n" +
+		"If no Lease ID is provided, uses the active lease for the requesting user. \n" +
+		"Sets AWS CLI credentials if used with no flags",
+	Args:  cobra.MaximumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
-		Service.LoginToLease(args[0], loginProfile, loginOpenBrowser, loginPrintCreds)
+		opts := &service.LeaseLoginOptions{
```

Comment:
moved login opts to it's own struct 
We're passing these same options around in a few places in the code,
so I like having them nicely packaged